main.go

```diff
@@ -56,9 +57,11 @@ func main() {
 
 	ctx := cli.ContextWithCancelOnSignal(context.Background())
 
+	vklog.L = vkklogv2.New(map[string]interface{}{"provider": "systemk"})
+
 	o, err := opts.FromEnv()
 	if err != nil {
-		klog.Fatal(err)
+		vklog.G(ctx).Fatal(err)
```

Comment:
can we not have to type `G(ctx)`, if find that API a bit ugly and would rather not see it, `klog.<level>` looks way nicer IMHO

Reply:
Yes, I agree. I was just experimenting sharing the VK logger but won't do that.